tests/output/webpage/output_test.js

```diff
@@ -263,24 +263,11 @@ describe("Linting", function() {
         ]
     );
 
-    if (!isFirefox()) {
-        // An exception occurs in slowparse when parsing this HTML on
-        // Chrome, Safari, and phantomjs.
-        failingTest("Fatal slowparse error detected",
-            "<li><a href='</li><img src='https://www.kasandbox.org'>", [
-                {row: 0, column: 0, lint: {type: "UNKNOWN_SLOWPARSE_ERROR"}}
-            ]
-        );
-    }
-
-    if (isFirefox()) {
-        // slowparse succeeds when parsing this HTML on Firefox.
-        failingTest("Not so fatal slowparse error detected (Firefox)",
-            "<li><a href='</li><img src='https://www.kasandbox.org'>", [
-                {row: 0, column: 7, lint: {type: "INVALID_ATTR_NAME"}}
-            ]
-        );
-    }
+    failingTest("Slowparse error detected",
```

Comment:
It used to be that Chrome and Firefox behaved differently, but they now appear to act the same (at least locally, let's see what TravisCI says)